### PR TITLE
IJSDK-674

### PR DESCRIPTION
**Pull request by JohnHake**
Please review the new gradle project configuration page for RubyMine.

Test plugin is at: https://github.com/JohnHake/scratch/tree/master/additional_products/rubymine_basics

### CODE REVIEW

File: products/rubymine.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Introduction
+RubyMine is an IntelliJ Platform-based product.
+Plugin projects for RubyMine can be developed using IntelliJ IDEA with the `gradle-intellij-plugin`.
+
+## Configuring Plugin Projects Targeting RubyMine
+The configuration of RubyMine plugin projects follows the methods described in [Configuring Plugin Projects using the IntelliJ IDEA Product Attribute](dev_alternate_products.md#configuring-plugin-projects-using-the-intellij-idea-product-attribute), and [Configuring the plugin.xml File](dev_alternate_products.md#configuring-pluginxml).
+
+The table below summarizes the `gradle-intellij-plugin` attributes to set in the `build.gradle` file for a RubyMine plugin project:
+
+| `gradle-intellij-plugin` Attribute | <br>Attribute Value |
```
Comment (YannCebron) [inline products/rubymine.md]: `<br>` = ?

File: products/rubymine.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+|-----------|-------|
+| [`intellij.type`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | `IU` for IntelliJ IDEA Ultimate.  |
+| [`intellij.version`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | Set to the same `IU` BRANCH.BUILD as the RubyMine target version, e.g. `192.7142.36` |
+| [`intellij.plugins`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | `org.jetbrains.plugins.ruby:2019.2.20191029` for the Ruby plugin.<br>See below for Ruby plugin version information. |
+| [`runIde.ideaDirectory`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#running-dsl) | Path to locally installed target version of RubyMine. For example, on macOS:<br>`/Users/<user name>/Library/Application Support/JetBrains/Toolbox/apps/RubyMine/ch-0/192.7142.37/RubyMine.app/Contents` |
+
+The required `org.jetbrains.plugins.go` plugin isn't compatible with IntelliJ IDEA Community edition but is compatible with IntelliJ IDEA Ultimate (`IU`) edition.
```
Comment (YannCebron) [inline products/rubymine.md]: ruby

File: products/rubymine.md (hunk 1 above)
Comment (JohnHake) [reply products/rubymine.md]: It's just formatting - a poor man's way to set the "Attribute Value" to bottom justify.

File: products/rubymine.md (hunk 2 above)
Comment (JohnHake) [reply products/rubymine.md]: Copy and paste error. (Apologies, you shouldn't have to catch that.)

**Pull request merged**